### ISSUE TITLE
Shapeless 2.0 support

### ISSUE THREAD

**Issue by davidhoyt** [gettyimages/spray-swagger#53]
Would you consider releasing a version against shapeless 2.0.0 similar to `"io.spray" %% "spray-routing-shapeless2" % "1.3.2"`?

**Comment by mhamrah** [comment]
Yes, it seems many people want to make the move to shapeless2. I'll look into this soon, but feel free to submit a PR for the dual build if it's a pressing issue for you.
